ansible/roles_ocp_workloads/ocp4_workload_rhel_worker/readme.adoc

```diff
@@ -0,0 +1,58 @@
+= ocp4-workload-rhel-worker - Set up AWS RHEL8 worker via tasks on Bastion
+
+== Role overview
+
+* This role installs playbooks and shell scripts to deploy a RHEL8 Worker for AWS based deployments.
+
+Once this role has been executed there is a directory `/opt/rhel_worker` with the necessary files to deploy a rhel_worker.
+If `ocp4_workloads_rhel_worker_deploy_worker` is set, this workload will deploy one worker.
+You can deploy subsequent workers with the playbook.
+SSH into the bastion, change to the `/opt/rhel_worker` directory and run `./setup_rhel_worker.sh`.
+This script will execute the playbook to deploy a rhel_worker into the region specified.
+
+Currently only `eu-central-1`, `us-east-1` and `us-east-2` are supported. This is because only for these three regions the AMI for the RHEL 8 image is specified.
```

Comment:
i see you're using image name and owner to find the image.

Isn't the image available in other regions?

Reply:
readme updated